packages/babel-parser/package.json

```diff
@@ -55,7 +55,8 @@
         "exports": {
           ".": {
             "require": "./lib/index.cjs",
-            "default": "./lib/index.js"
+            "default": "./lib/index.js",
+            "types": "./typings/babel-parser.d.ts"
```

Comment:
I'm very surprised that `types` after `default` works

Reply:
I finally understand what you mean now, and yes I think it would be better to follow the documentation of ts.
https://www.typescriptlang.org/docs/handbook/esm-node.html#packagejson-exports-imports-and-self-referencing

Reply:
Really weird here, CI started to fail after I moved `types` to the front.